internal/cluster/cluster_operations.go

```diff
@@ -372,7 +373,7 @@ func FlattenCluster(ctx context.Context, clusterObj *models.Cluster, apiClient *
 	result["is_default"] = clusterObj.IsDefault
 	result["is_stretched"] = clusterObj.IsStretched
 
-	flattenedVdsSpecs := getFlattenedVdsSpecsForRefs(clusterObj.VdsSpecs)
+	flattenedVdsSpecs, err := getFlattenedVdsSpecsForRefs(ctx, clusterObj.ID, apiClient)
```

Comment:
```suggestion
	flattenedVdsSpecs, err := getFlattenedVdsSpecsForRefs(ctx, clusterObj.ID, apiClient)
	if err != nil {
    	return nil, err
	}
```

Reply:
Otherwise you will get an "ineffectual assignment to err (ineffassign)"

Reply:
i thin i got this in the last comment